data/profiles/default.yaml

```diff
@@ -131,6 +131,16 @@ spec:
             rollingUpdate:
               maxSurge: "100%"
               maxUnavailable: "25%"
+      tracing:
+        enabled: false
+      kiali:
+        enabled: false
+      grafana:
+        enabled: false
+      prometheus:
```

Comment:
do we really want to disable prometheus for default profile?

Reply:
according to doc, we turned prometheus on for default profile. Will change this.